### ISSUE TITLE
culture parameter being ignored in razor component (Blazor WebAssembly)

### ISSUE THREAD

**Issue by cyl1d3** [Humanizr/Humanizer#1089]
I'm currently having the following issue:

I'm using Humanizer in a razor component running in a WebAssembly Blazor project. My component has a class `Localizer` which provides a CultureInfo. When passing a DateTime through the Humanizer, I always get the English result. Here's the code snippet:

``` 
<small class="pr-2">
    <abbr title="@item.Date.ToString(Localizer.GetCulture())">
        @item.Date.Humanize(culture: Localizer.GetCulture())
    </abbr>
</small>
```

The Localizer is not the problem, as the Date format in the tooltip is correctly changed when I change the culture:

Culture en-US:
![image](https://user-images.githubusercontent.com/1682435/122439102-a0d31280-cf9b-11eb-8332-d67830308c6c.png)

Culture de-DE:
![image](https://user-images.githubusercontent.com/1682435/122439170-b5afa600-cf9b-11eb-9dec-e26e3881c2de.png)

**Comment by Velociapcior** [comment]
hello @cyl1d3 I had the same problem, I solved it by adding boolean 'false' value before culture, as this parameter corresponds to bool? utcDate = null (from decompiling dll). It solved my issue, you could give it a try. I hope it will help you

**Comment by cyl1d3** [comment]
@Velociapcior Thanks for the hint! Sadly, this did not work for me.

**Comment by zuverza** [comment]
@cyl1d3 I don't think that feature is implemented yet. I was looking at [DateToOrdinalWordsConverterRegistry.cs](https://github.com/Humanizr/Humanizer/blob/v2.11.10/src/Humanizer/Configuration/DateToOrdinalWordsConverterRegistry.cs) and it uses the DefaultDateToOrdinalWordConverter() for 'de'.

**Comment by riccardominato** [comment]
I'm having exactly the same issue here.

I'm trying to humanize a `TimeSpan`.

Calling the same method with same parameters inside an Asp.Net Core (backend) project works perfectly, while in a Blazor project the Spanish language doesn't work and it shows the result in english.
Oddly, Italian (my native language) works.

As for @cyl1d3, if I try something not related to TimeSpans it works in Spanish, too.

This is the instruction I'm trying to perform.

```csharp
TimeSpan.FromMinutes(20).Humanize(culture: new CultureInfo("es"))
```
Using `es-ES` doesn't change the result.

`Humanize` package is installed in my project, so I should have all languages available.

@cyl1d3 I know it's been a long time since you opened this issue, but did you find a workaround?

Thank you.

**Comment by riccardominato** [comment]
I found out what was the problem! It seems that Humanizer in Blazor only support English and the browser current language.

Since in my case it was set in Italian, it worked with Italian and English.
I tried setting my browser language in Spanish and magically now it works for Spanish humanization (but of course not for Italian anymore).

**Comment by rasmus-carlsson** [comment]
I'm having the same issue
